### PR TITLE
First setup

### PR DESCRIPTION
**Pull request by tmampa**
# In this PR:
<br>

- [ ] Iniatilzed Webpack and configured it
- [ ] Add UI to the app
- [ ] Style the app
- [ ] Use JS to display score

## Changes implemented:
<br>

- [ ] Change display function to an  arrow function (ES6)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{
+  name: 'Dave',
+  score: 100,
+},
+{
+  name: 'Tshephang',
+  score: 100,
+}];
+
+function displayScores() {
```
Comment (leolpaz) [inline src/index.js]: - [x] Great job with the js, but one of the learning objectives for this project is to use ES6 syntax, which means that `function displayScores() {...}` should be an arrow function as in `const displayScores = () => {...}`

**Review by leolpaz — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

You've done an amazing job so far, I'd like to highlight the following:
- Great job with displaying the scores
- Good README file

Some issues need to be resolved before your approval, but you're almost there!


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (tmampa) [reply src/index.js]: Thank you I implemented the change

Comment (rloterh) [reply src/index.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Tshephang, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**